mrp_production_project_estimated_cost/wizard/wiz_create_fictitious_of.py

```diff
@@ -34,15 +35,16 @@ def do_create_fictitious_of(self):
                     'location_src_id': production_obj._src_id_default(),
                     'location_dest_id': production_obj._dest_id_default(),
                     'active': False,
-                    'product_uom': product.uom_id.id
+                    'product_uom': product.uom_id.id,
+                    'project_id': self.project_id.id
                     }
             new_production = production_obj.create(vals)
+            new_production.action_compute()
```

Comment:
Esto es un asistente que crea las OF ficticias. Que son ordenes de producción inactivas que no afectan a otra cosa que no sea calcular costes para un producto en concreto.
Este asistente, se lanza desde el listado de productos o de plantillas de productos, y partiendo de esos productos, se crea una OF por cada. 
Ejecuto el action_compute después de crear la OF, para que arrastre datos como listas de materiales y demás, no hay ningún paso manual entre medias, ya que la ejecución del compute es inmediata a la creación. Si que hay una opción que se elige en el asistente, que además de crear la OF, te calcula los costes estimados y te carga el coste en la ficha de producto. 
Realmente no entiendo que problema hay en ejecutar el action_compute, si los pasos que se ejecutan son automáticos y no hay pasos manuales. Si quieren hacer los cambios, los pueden hacer después de que se cree y se hayan calculado los datos.
Igual es que estoy muy ofuscada y no entiendo lo que me quieres decir.